apps/server/src/modules/authentication/uc/account.uc.spec.ts

```diff
@@ -624,30 +658,6 @@ describe('AccountUc', () => {
 			);
 			expect(accounts.data).toStrictEqual([]);
 		});
-		it('should throw, if skip is smaller than 0', async () => {
```

Comment:
Why was that removed?

Reply:
Cedric pointed out, that the validation of these values is already done via ValidationApi and it is redundant to do it again in the UC. So I removed the validation code from the UC and the tests checking it.